src/components/OneMonsterItem.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export default function OneMonsterItem({ monster }) {
+  return (
+    <ContainerStyled>
+      <MonsterNameStyled>{monster.name}</MonsterNameStyled>
+      <ContainerTypeCR>
```

Comment:
what means CR in ContainerTypye CR?

Reply:
Challenge-Rating: Its a Stat provided by the object "monster". This container includes the information for the props "type" and "challenge-rating".

Maybe i should rename the ContainerTypeCR to "ContainerForTypeAndCR" ?